src/main/java/com/synopsys/integration/properties/TestPropertiesManager.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * integration-common
+ *
+ * Copyright (c) 2024 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.properties;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+
+import org.junit.jupiter.api.Assumptions;
+
+import com.synopsys.integration.exception.IntegrationException;
+
+/**
+ * Create Properties object from a Properties object, a file, the environment,
+ * or both a file and the environment.
+ * Intended to be used for Unit tests.
+ *
+ * @author danam
+ * @since April 19, 2024
+ */
+public class TestPropertiesManager extends PropertiesManager {
+    public static final String DEFAULT_TEST_PROPERTIES_FILE_NAME = "test.properties";
+    public static final String DEFAULT_TEST_PROPERTIES_LOCATION = "src/main/resources/" + DEFAULT_TEST_PROPERTIES_FILE_NAME;
+
+    /**
+     * Load properties from existing Properties.
+     *
+     * @param properties
+     *            An existing Properties object.
+     * @return TestPropertiesManager
+     */
+    public static TestPropertiesManager loadProperties(Properties properties) {
+        return new TestPropertiesManager(properties);
+    }
+
+    /**
+     * Load properties from <b>default</b> test file only: {@value DEFAULT_TEST_PROPERTIES_LOCATION}
+     * Validation is performed against the properties file, but not it's contents.
+     * No attempt to load the file will be made if it is invalid.
+     *
+     * @return TestPropertiesManager
+     */
+    public static TestPropertiesManager loadFromDefaultFile() {
+        return TestPropertiesManager.loadWithOverrides(DEFAULT_TEST_PROPERTIES_LOCATION, Collections.emptyMap());
```

Comment:
Can we use Map.of() here it creates an immutable empty map.  Not going to prevent approving the PR for this. 
```suggestion
        return TestPropertiesManager.loadWithOverrides(DEFAULT_TEST_PROPERTIES_LOCATION, Map.of());
```

Reply:
Unable to do this edit as this has a language level of Java 8, and Map.of() was introduced in Java 9.